src/app/test-record/test-record.module.ts

```diff
@@ -0,0 +1,69 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { RouterModule } from '@angular/router';
+import { MaterialModule } from '@app/material.module';
+import { PendingChangesGuard } from '@app/shared/pending-changes-guard/pending-changes.guard';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faGithub, faMedium, faStackOverflow } from '@fortawesome/free-brands-svg-icons';
+import { faCheckSquare as farCheckSquare, faSquare as farSquare } from '@fortawesome/free-regular-svg-icons';
+import { faCheckSquare, faCoffee, faMinus, faPlus, faSquare } from '@fortawesome/free-solid-svg-icons';
+import { AuthenticationGuard } from 'microsoft-adal-angular6';
+import { NgrxFormsModule } from 'ngrx-forms';
+import { NgxJsonViewerModule } from 'ngx-json-viewer';
+import { SharedModule } from '../shared/shared.module';
+import {TestRecordComponent} from '@app/test-record/test-record.component';
+import { VehicleComponent } from './vehicle/vehicle.component';
+import { TestSectionComponent } from './test-section/test-section.component';
+import { DefectsComponent } from './defects/defects.component';
+import { SeatbeltInstallationCheckComponent } from './seatbelt-installation-check/seatbelt-installation-check.component';
+import { EmissionDetailsComponent } from './emission-details/emission-details.component';
+import { VisitComponent } from './visit/visit.component';
+import { NotesComponent } from './notes/notes.component';
+import { TestHistoryComponent } from './test-history/test-history.component';
+
+@NgModule({
+  imports: [
```

Comment:
Please recheck this module. Remove lines that are not needing for the module to work